app/uk/gov/hmrc/agentclientrelationshipsfrontend/models/client/ClientExitType.scala

```diff
@@ -22,6 +22,6 @@ enum ClientExitType(val name: String):
   case CannotFindAuthorisationRequest extends ClientExitType("cannot-find-authorisation-request")
   case AuthorisationRequestExpired extends ClientExitType("authorisation-request-expired")
   case AuthorisationRequestCancelled extends ClientExitType("authorisation-request-cancelled")
-  case AlreadyRespondedToAuthorisationRequest extends ClientExitType("authorisation-request-already-responded-to")
+  case AlreadyRespondedToAuthorisationRequest extends ClientExitType("authorisation-request-already-accepted")
```

Comment:
this exit type is currently used when the invitation has that status Rejected https://github.com/hmrc/agent-client-relationships-frontend/blob/70c59e8907e70862e785d4471b94109348d83063/app/uk/gov/hmrc/agentclientrelationshipsfrontend/controllers/client/ConsentInformationController.scala#L69 this new content does not support both